src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -61,6 +61,17 @@ export module DotNet {
     0: new JSObject(window, 0),
   };
 
+  cachedJSObjectsById[0]._cachedFunctions.set('import', (url: any) => {
+    // In most cases developers will want to resolve dynamic imports relative to the base HREF.
+    // However since we're the one calling the import keyword, they would be resolved relative to
+    // this framework bundle URL. Fix this by providing an absolute URL.
+    if (typeof url === 'string' && url.startsWith('./')) {
+      url = document.baseURI + url.substr(2);
```

Comment:
I did consider auto-prefixing regular base-relative URIs (e.g., `myfile.js`) with the base URI, but am reluctant to do that because we can't predict other possible syntaxes that browsers might add in the future.

However it is safe to rewrite URLs starting with `./` because they are unambiguous - we know for sure what that means, and there's no use case for `./` meaning relative to anything other than the base URI since at runtime the component itself doesn't have any location, and all our other URL-resolution behaviors are base relative too.

Reply:
I'm not sure what this bit means, can you give an explicit example?